dipy/viz/ui.py

```diff
@@ -28,16 +28,10 @@ class UI(object):
 
     Attributes
     ----------
-    ui_param : object
-        This is an attribute that can be passed to the UI object by the
-        interactor.
-    ui_list : list of :class:`UI`
-        This is used when there are more than one UI elements inside
-        a UI element. They're all automatically added to the renderer at the
-        same time as this one.
-    parent_ui: UI
-        Reference to the parent UI element. This is useful of there is a parent
-        UI element and its reference needs to be passed down to the child.
+    position : (float, float)
+        Absolute coordinates (x, y) in pixels the lower-left corner.
```

Comment:
'of the'? In fact, you already wrote a more complete sentence below `Absolute coordinates (x, y) of the lower-left corner of this UI component.` It would be better to settle for one!